test/router-luatest/router_test.lua

```diff
@@ -545,3 +545,41 @@ g.test_enable_disable = function(g)
     -- we don't want this server to interfere with subsequent tests
     g.router_1:drop()
```

Comment:
I would omit this part *"However, from 2.10.1 worker fiber is invincible and cannot be killed at all"* from the commit message until the fix is actually submitted. We can't rely on it until it is really done.